src/components/ClaimedBooks/ClaimedBooks.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
```

Comment:
This import isn't being used and should be removed.

Reply:
e6cf020